azuredeploy.json

```diff
@@ -24,12 +24,68 @@
             "type": "bool"
         },
         "azureBackupSwitch": {
-            "defaultValue": false,
+            "defaultValue": true,
```

Comment:
Any reason for this default value change? I know it was true before and I changed it to false in that we need to make the defaults as simple/small as possible. In that regard, we may even need to change the redisDeploySwitch default to false as well, and we'll handle that separately. If there's no special reason to revert this back to true, can we change this back to false?